WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -639,4 +630,19 @@ public void onPostUploaded(OnPostUploaded event) {
             loadPosts(LoadMode.FORCED);
         }
     }
+
+    /*
+     * Media info for a post's featured image has been downloaded, tell
+     * the adapter so it can show the featured image now that we have its URL
+     */
+    @SuppressWarnings("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onMediaChanged(MediaStore.OnMediaChanged event) {
+        if (isAdded() && !event.isError() && getPostListAdapter() != null) {
```

Comment:
You're right. I just used the previous implementation and didn't really think about it, which is my bad. Fixed in 39163f0892929902f88e6808a06694836a1a26ab